src/a_star.jl

```diff
@@ -0,0 +1,93 @@
+"""
+    get_distance(A::Int, B::Int, 
+					nodes::Dict{Int,T} , 
```

Comment:
please align the code properly and do not use over 92 chars per line.

Reply:
Corrected, I hope now it's ok.